code/modules/mob/mob_movement.dm

```diff
@@ -175,6 +175,9 @@
 	if(mob && .)
 		mob.throwing = 0
 
+	for(var/obj/O in mob)
+		O.on_mob_move(direct, src)
```

Comment:
By the way @phil235 @ChangelingRain this really indicates we need a way to hook proc's like this., so you only have to pay the price for objects that actually need the functionality - at the cost of extra bookkeeping.

anyway it's irrelevant to this pr.